Type-annotate serialization.py

malfet opened the pull request:
Move Storage class from __init__.pyi.in to types.py and make it a protocol, since this is not a real class
Expose `PyTorchFileReader` and `PyTorchFileWriter` native classes

Ignore function attributes, as there are yet no good way to type annotate those, see https://github.com/python/mypy/issues/2087

malfet commented:
Protocol is Python-3.8 feature, although mypy still understands it

ezyang reviewed (approved):
You have a bunch of spurious submodule updates

malfet committed:
Type-annotate serialization.py

Move Storage class from __init__.pyi.in to types.py
(It should be a protocol class, but protocols are Pyton3.8 feature)
Expose `PyTorchFileReader` and `PyTorchFileWriter` native classes

Ignore function attributes, as there are yet no good way to type annotate those, see https://github.com/python/mypy/issues/2087
